llvm/lib/Target/SystemZ/SystemZTargetTransformInfo.cpp

```diff
@@ -1244,17 +1247,43 @@ InstructionCost SystemZTTIImpl::getInterleavedMemoryOpCost(
   return NumVectorMemOps + NumPermutes;
 }
 
-static int getVectorIntrinsicInstrCost(Intrinsic::ID ID, Type *RetTy) {
+static int getVectorIntrinsicInstrCost(Intrinsic::ID ID, Type *RetTy, const SmallVectorImpl<Type *>& ParamTys) {
   if (RetTy->isVectorTy() && ID == Intrinsic::bswap)
     return getNumVectorRegs(RetTy); // VPERM
-  return -1;
+  else if (ID == Intrinsic::vector_reduce_add) {
+    // Retrieve number and size of elements for the vector op
+    auto *VTy = cast<FixedVectorType>(ParamTys.front());
+    unsigned numElements = VTy->getNumElements();
```

Comment:
LLVM has some specific coding guidlines: https://llvm.org/docs/CodingStandards.html

For one thing, variable names should always start with a capital letter.

Reply:
rtfm, got it :sweat_smile: 
will read and fix as needed,

Reply:
:)  I only noticed the missing capital first letter here, but mentioned the document as it is good to be aware of...

Reply:
done, i think ;)